coss/opensource_clubs/templates/opensource_clubs/home_page.html

```diff
@@ -9,66 +9,52 @@
-        <h2>
-          {{ page.title }}
-        </h2>
-        {{ page.description|richtext }}
+        {% include 'home/blocks/home_heading.html' %}
       </div>
-      <div class="col-md-7 text-right">
-        <img class="img-fluid" src="{% static 'img/clubs-illustration.svg' %}" alt="Open Source Clubs">
+      <div class="col-md-7 text-right hidden-sm-down">
+        <img class="img-fluid" src="{% static 'img/laptop-illustration.svg' %}" alt="Open Source Clubs">
       </div>
+    </div>
   </div>
 {% endblock %}
 
 {% block content %}
-  <div class="container content">
-    {% if page.discourse %}
-      <div class="row">
-        <div class="col">
-          <h6>Discourse Category: {{ page.discourse.name }}</h6>
+  <div class="content">
+    <div class="container">
+      {% if page.discourse %}
+        <div class="row">
+          <div class="col">
+            <h6>Discourse Category: {{ page.discourse.name }}</h6>
+          </div>
         </div>
-      </div>
-    {% endif %}
-
-    <div class="row">
-      <div class="col-md-6">
-        <h4>Features Clubs</h4>
-      </div>
-      <div class="col-md-6 text-right">
-        <a href="#" class="btn btn-outline-primary">Register a Club</a>
-      </div>
-    </div>
+      {% endif %}
 
-    {% if page.get_featured %}
-      <div class="card-deck">
-        {% for club in page.get_featured %}
-          {% include 'opensource_clubs/club_card.html' with club=club.specific %}
-        {% endfor %}
-      </div>
-    {% endif %}
-
-    {% if page.get_category_page %}
-      <div class="text-center mt-4">
-        <a href="{% pageurl page.get_category_page %}">
-          See All Of Our Clubs
-          <img class="ml-2" src="{% static 'img/icons/right-arrow-black.svg' %}" alt="more">
-        </a>
-      </div>
-    {% endif %}
-
-    {% if page.testimonial %}
-      <div class="row testimonial">
-        <div class="col-sm-2">
-          {% image page.testimonial.photo width-400 class="img-fluid" %}
+      {% if page.get_featured %}
+        <div class="card-deck">
+          {% for club in page.get_featured %}
+            {% include 'opensource_clubs/club_card.html' with club=club.specific %}
+          {% endfor %}
         </div>
-        <div class="col-sm-10">
-          <blockquote>
-            {{ page.testimonial.quote|richtext }}
-            <p class="quote-person">{{ page.testimonial.person }}</p>
-          </blockquote>
+      {% endif %}
+
+      {% if page.get_category_page %}
+        <div class="text-center mt-4">
+          <a href="{% pageurl page.get_category_page %}">
+            See All Of Our Clubs
+            <img class="ml-2" src="{% static 'img/icons/right-arrow-black.svg' %}" alt="more">
+          </a>
         </div>
-      </div>
+      {% endif %}
+
+      {% if page.testimonial %}
+        {% include 'home/blocks/testimonial.html' with testimonial=page.testimonial %}
+      {% endif %}
+    </div>
+
+    {% if page.bottom_cta_title %}
```

Comment:
The bottom cta and the footer have a big space in between.
![screenshot-2017-9-11 homepage](https://user-images.githubusercontent.com/900565/30272245-417da10a-96fc-11e7-91a4-67be41b935ae.png)